### DISCUSSION TITLE
Listen on a public IP

### DISCUSSION THREAD

**Discussion by Hanan-Natan** [mitmproxy/mitmproxy#7380] [General]
Hi,
I want to intercept the traffic of a specific software that requires me to add a public proxy IP.
I would like to intercept the traffic so I thought about installing mitmproxy on a VPS with public IP.
Would that work? Is it possible to use mitmproxy in such scenario?

Thank you!

**Comment by blockchainexpert2022** [comment]
No, you cannot do that.

**Comment by mhils** [comment]
Sure - you can absolutely run mitmproxy on a public IP:

- https://docs.mitmproxy.org/stable/concepts-options/#proxyauth
- https://docs.mitmproxy.org/stable/concepts-options/#block_global
